Makefile

```diff
@@ -206,7 +206,7 @@ format:
 fmt: format
 
 lint:
-	if [ $$(wc -l < tests/utils.go) -gt 2813 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
+	if [ $$(wc -l < tests/utils.go) -gt 2350 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
```

Comment:
Can I ask you how is this value calculated? Thanks

Reply:
Its simply the number of lines in tests/utils.go after the move of the functions

Reply:
:man_facepalming: I was looking at the previous version file! That's why something wasn't right! Thanks, sorry for the noise